### PR TITLE
Add Iris Flower & Include ML Flow Integrations

### PR DESCRIPTION
**Pull request by sayonsivakumaran**

### CODE REVIEW

File: run.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-    - batch_size: [1, 2]
-      epoch: [1]
-      learning_rate: [0.001, 0.01]
\ No newline at end of file
+    - alpha: [0.01]
+      max_iter: [100]
+      MLFLOW_TRACKING_URI: 'http://mlflow.gcp.impact.georgian.io'
+      MLFLOW_TRACKING_USERNAME: 'gcp-hydra-user'
+      MLFLOW_TRACKING_PASSWORD: 'ayupD3kTsxfps'
+      GOOGLE_APPLICATION_CREDENTIALS: './credentials/hydra-gcp-test-291317-81faaa44d19d.json'
```
Comment (coder46) [inline run.yaml]: Remove line 14

Comment (coder46) [inline run.yaml]: Remove line 14 as this a local run specific detail

File: run.yaml [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-      learning_rate: 0.0001
-    - batch_size: [1, 2]
-      epoch: [1]
-      learning_rate: [0.001, 0.01]
\ No newline at end of file
+    - alpha: [0.01]
+      max_iter: [100]
+      MLFLOW_TRACKING_URI: 'http://mlflow.gcp.impact.georgian.io'
+      MLFLOW_TRACKING_USERNAME: 'gcp-hydra-user'
+      MLFLOW_TRACKING_PASSWORD: 'ayupD3kTsxfps'
```
Comment (coder46) [inline run.yaml]: Also lets mask the username and password here and replace with <enter_user_name> <enter_password> placeholders for security reasons as this is a public repo. We will also need to refresh the password
